helpers/swagger.js

```diff
@@ -0,0 +1,86 @@
+var jsonRefs = require('json-refs');
+var YAML = require('js-yaml');
+var path = require('path');
+var fs = require('fs');
+var	ZSchema = require('./z_schema');
```

Comment:
white space here

Reply:
There is no issue in spaces in the code. I guess somehow github is messing up. Don't know exactly. 

![screenshot at nov 03 11-51-12](https://user-images.githubusercontent.com/112468/32370432-50fd050c-c08d-11e7-8688-b1aea1d775a5.png)